maquetado/index.html

```diff
@@ -128,6 +128,13 @@ <h3 class="m-bottom-1">Django</h3>
           </article>
         </div>
       </section>
+      <footer class="footer-project">
```

Comment:
same as the previous PR, try to let `<nav>` and `<footer>` in the same level of the `<body>`